runconfig/config_windows.go

```diff
@@ -0,0 +1,13 @@
+package runconfig
+
+// ContainerConfigWrapper is a Config wrapper that hold the container Config (portable)
+// and the corresponding HostConfig (non-portable).
+type ContainerConfigWrapper struct {
+	*Config
+	HostConfig *HostConfig `json:"HostConfig,omitempty"`
```

Comment:
I'd say keep it InnerHostConfig to make it obvious that it's the same one than in Linux even if both are tagged as HostConfig.